### PR TITLE
Fix for #1 to allow specification of index and jira datetimezones

### PR DESCRIPTION
**Pull request by Anusien**

### CODE REVIEW

File: src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java [hunk 1]
```diff
@@ -132,17 +134,16 @@ private String getBasicInfoURL() throws UnsupportedEncodingException {
         return url;
     }
 
-    protected static final DateTimeZone JIRA_TIME_ZONE = DateTimeZone.forID("America/Chicago");
     private static final DateTimeFormatter JIRA_TIME_FORMAT = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm");
     /**
      * Imhotep builders always runs in UTC-6, regardless of DST. However, Jira observes daylight savings time.
```
Comment (youknowjack) [inline src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java]: This comment is now incorrect. I think there's a way to rephrase to say that it is not recommended that Imhotep builders observe DST.

File: src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java [hunk 2]
```diff
@@ -132,17 +134,16 @@ private String getBasicInfoURL() throws UnsupportedEncodingException {
...
      * So when we're in DST and run for "2018-04-01" to "2018-04-02", we need to tell Jira we actually mean
```
Comment (youknowjack) [inline src/main/java/com/indeed/jiraactions/api/IssuesAPICaller.java]: Can you add a unit test case for this? (maybe add back in the deleted TestsIssuesApiCaller?)

**Comment by kevindigo** [comment]
Should we abandon this PR, or is there still hope for it?

**Comment by Anusien** [comment]
We shouldn't abandon it.

**Pull request closed without merging**